ajax/libs/startbootstrap-freelancer/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "startbootstrap-freelancer",
   "filename": "js/freelancer.min.js",
-  "version": "3.3.7",
+  "version": "4.0.0",
```

Comment:
Thanks for your contribution, however, we can not directly bump the version number, if the auto-update didn't fetch the new versions properly, we should try to dig about the root cause, usually to update the auto-update config. If we really need to add the new version manually, we should also commit the assets.

Reply:
@PeterDaveHello There are 3 versions 4.0.0 on npm (alpha, beta and release). Should we add all of the manually or only 4.0.0 (release)?
Also may be we can ask author to add git tag of v4.0.0 to git repo and do nothing :)